packages/tupaia-web/src/views/ProjectsModal.tsx

```diff
@@ -54,19 +56,33 @@ const ExploreButton = styled(RouterButton).attrs({
   to: DEFAULT_URL,
 })`
   margin-bottom: 16px;
-  width: 250px;
-  height: 50px;
+  margin-left: 0.4rem;
+  width: 200px;
+  height: 40px;
   border-radius: 3px;
-  font-size: 13px;
+  font-size: 14px;
```

Comment:
We should probably use Rem here too

Reply:
Also I think there are a few tiny design tweaks to make:
- the font-weight of the Projects text
- the styles of the intro text
- the Explore Tupaia.org button outline color and font-weight